examples/jsm/exporters/GLTFExporter.js

```diff
@@ -718,6 +721,8 @@ class GLTFWriter {
 
 		if ( metalnessMap === roughnessMap ) return metalnessMap;
 
+		// if ( occlusion === roughness && roughness === metalness ) return material.aoMap;
```

Comment:
Not sure what to do here. 😕

Reply:
Not sure either, I removed the comment for now again